Dockerfile.ubuntu

```
@@ -0,0 +1,8 @@
+FROM ubuntu:bionic
+MAINTAINER sig-platform@spinnaker.io
+RUN apt-get update && apt-get -y install openjdk-11-jre-headless wget
+RUN adduser --disabled-login --system spinnaker
```

```suggestion
RUN adduser --system --uid 10111 --group spinnaker
```

(same uid change, plus `--disabled-login` is redundant, it's implied by `--system`)